src/ca65/instr.c

```diff
@@ -783,7 +788,7 @@ static const InsTable* InsTabs[CPU_COUNT] = {
     (const InsTable*) &InsTab65816,
     (const InsTable*) &InsTabSweet16,
     (const InsTable*) &InsTabHuC6280,
-    0,                                  /* Mitsubishi 740 */
+    0,
```

Comment:
* On Fri, Jul 24, 2015 at 12:33:41PM -0700 Christian Groessler wrote:
> In src/ca65/instr.c:
> 
> > @@ -783,7 +788,7 @@ static const InsTable* InsTabs[CPU_COUNT] = {
> >      (const InsTable*) &InsTab65816,
> >      (const InsTable*) &InsTabSweet16,
> >      (const InsTable*) &InsTabHuC6280,
> > -    0,                                  /* Mitsubishi 740 */
> > +    0,
> 
> But ca65 doesn't seem to support a "Misubishi 740" cpu...

The Mitsubishi 740 seems to be very similar to the 6502:

http://www.hifi-remote.com/files/chip-manuals/Mits-740-sum.pdf

I believe that Uz once said that he made a special version of ca65
(under NDA) for a special processor. Perhaps, it was the Mitsubishi 740,
and the comment above is a leftover of that?

Regards,
Spiro.

-- 
Spiro R. Trikaliotis
http://www.trikaliotis.net/